pinot-hadoop/src/main/java/org/apache/pinot/hadoop/job/SegmentPreprocessingJob.java

```diff
@@ -375,13 +377,37 @@ private void addHashCodeField(Set<Schema.Field> fieldSet) {
     fieldSet.add(hashCodeField);
   }
 
+  protected org.apache.pinot.common.data.Schema getSchema()
```

Comment:
Use the one in Base class, remove the duplicate code in this class

Reply:
same as above